documentation/src/main/asciidoc/user_guide/transactions.adoc

```diff
@@ -235,34 +235,15 @@ That is because the optimistic transactions rollback if data has changed between
 On the other hand, pessimistic transactions might be a better fit when there is high contention on the keys and transaction rollbacks are less desirable.
 Pessimistic transactions are more costly by their nature: each write operation potentially involves a RPC for lock acquisition.
 
-=== Write Skew [[tx:write-skew]]
+=== Write Skews [[tx:write-skew]]
 
-The write skew anomaly occurs when 2 transactions read and update the same key and both of them can commit successfully without having seen the update performed by the other.
-To detect and rollback one of the transaction, `write-skew` should be enabled.
+Write skews occur when two transactions independently and simultaneously read and write to the same key. The result of a write skew is that both transactions successfully commit updates to the same key but with different values.
 
-NOTE: The write skew check is only performed for `REPEATABLE_READ` isolation.
+Enabling `write-skew` in the {brandname} configuration automatically detects and rolls back one of the transactions. However, write skew checks apply to `REPEATABLE_READ` isolation only.
```

Comment:
again, enabled by default

Reply:
@rvansa from the user guide: "write-skew - enables the write skew check. Check section Write Skew for more details. Default is false."

http://infinispan.org/docs/stable/user_guide/user_guide.html#tx:configuration

looking at the schema docs, the default is true. so it looks like docs need to be updated here. schema docs also say write skew is deprecated. is that the case?